src/guide/mixins.md

```diff
@@ -2,7 +2,7 @@
 
 ## Basics
 
-Mixins are a flexible way to distribute reusable functionalities for Vue components. A mixin object can contain any component options. When a component uses a mixin, all options in the mixin will be "mixed" into the component's own options.
+Mixins are a simple way to distribute reusable functionalities for Vue components. A mixin object can contain any component options. When a component uses a mixin, all options in the mixin will be "mixed" into the component's own options.
```

Comment:
This is a good point, however we avoid the use of words like "simple" as they can be hostile to people learning (see our writing guide for more info). It might be worth just taking out the qualifier altogether here.

Reply:
```suggestion
Mixins distribute reusable functionalities for Vue components. A mixin object can contain any component options. When a component uses a mixin, all options in the mixin will be "mixed" into the component's own options.
```

What do you think about this change?